tests/test_tablepyxl.py

```diff
@@ -66,6 +66,17 @@
                "</tbody> " \
                "</table>"
 
+table_whitespace = """
+    <table name='whitespace table'>
+        <thead></thead>"
+        <tbody>
+            <tr>
+                <td>   a  bc  <inn1>  d  ef  </inn1>  g  hi   <inn2>   j  k  <inn3>  l  m  </inn3>  n  o  </inn2>  p  </td>
+            </tr>
+        </tbody>
+    </table>
+"""
+
```

Comment:
I have this new html test table listed using triple quotes because I think it's easier / more readable, but this is different than the above tables.  I either want to change the rest as part of this if we like it, or I'll change it back if not.  Just let me know.